packages/tooltip/src/Tooltip.tsx

```diff
@@ -380,17 +380,14 @@ function Tooltip({
       });
     }
 
-    return React.cloneElement(trigger, {
-      ...createTriggerProps(triggerEvent, trigger.props),
-      'aria-describedby': tooltipId,
-      children: (
-        <>
-          {trigger.props.children}
-          {tooltip}
-        </>
-      ),
-      className: cx(positionRelative, trigger.props.className),
-    });
+    return (
+      <div className={positionRelative} {...createTriggerProps(triggerEvent)}>
```

Comment:
In the past, we purposefully didn't do this, and instead made sure that the button DOM structure supported Tooltips. I'm still of the mind that we shouldn't wrap the trigger element.

Reply:
Looks like you reviewed this PR after I closed it. Please see https://github.com/mongodb/leafygreen-ui/pull/732 instead :)